app/views/articles/edit.html.erb

```diff
@@ -1,5 +1,5 @@
-<h1>Edit article</h1>
+<div class="container">
+  <h1>Edit article</h1>
  
-<%= render 'form' %>
- 
-<%= link_to 'Cancel', articles_path %>
\ No newline at end of file
+  <%= render 'form' %>
+</div>
```

Comment:
@NitinNROR always leave a new line at the bottom of a file before committing then this red icon will not display. Check all files for the new line if there...

Reply:
Done.